CONTRIBUTING.md

```diff
@@ -1,101 +1,40 @@
 # How to Contribute
 
-The Deis project is Apache 2.0 licensed and accepts contributions via Github pull
-requests. This document outlines some of the conventions on commit message formatting,
-contact points for developers and other resources to make getting your contribution
-accepted.
+Deis components are Apache 2.0 licensed and accept contributions via Github pull requests. This document outlines resources and conventions useful for anyone wishing to contribute either by reporting issues or submitting pull requests.
 
 # Certificate of Origin
 
-By contributing to this project you agree to the
-[Developer Certificate of Origin (DCO)][dco]. This document was created by the Linux
-Kernel community and is a simple statement that you, as a contributor, have the legal
-right to make the contribution.
+By contributing to any Deis project you agree to its [Developer Certificate of Origin (DCO)][dco]. This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution.
 
 # Support Channels
 
-Before opening a new issue, it's helpful to search the project - it's likely that another user
-has already reported the issue you're facing, or it's a known issue that we're already aware of.
+Before opening a new issue or PR against any Deis project, it's helpful to search that project's issue queue - it's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
 
-Additionally, see the [Troubleshooting Deis][troubleshooting] documentation for common issues.
+Additionally, see the [Troubleshooting][] documentation for common issues.
 
 Our official support channels are:
 
-- GitHub issues: https://github.com/deis/deis/issues/new
+- GitHub issue queues:
```

Comment:
I'll add that.  I think that's useful.  But I think the list is also useful since if someone wants to _open_ an issue, they need to do it from within the right repo.